internal/command/auth/clusters.go

```diff
@@ -0,0 +1,89 @@
+package auth
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/yaml"
+
+	"github.com/jetstack/jsctl/internal/client"
+	"github.com/jetstack/jsctl/internal/cluster"
+	"github.com/jetstack/jsctl/internal/command/types"
+	"github.com/jetstack/jsctl/internal/config"
+)
+
+func Clusters(run types.RunFunc, apiURL string) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:  "clusters",
+		Args: cobra.ExactArgs(0),
+	}
+
+	cmd.AddCommand(createServiceAccount(run, apiURL))
+
+	return cmd
+}
+
+func createServiceAccount(run types.RunFunc, apiURL string) *cobra.Command {
+	var serviceAccountFormat string
+
+	cmd := &cobra.Command{
+		Use:   "create-service-account [name]",
+		Short: "Create a new service account identity for a cluster",
```

Comment:
Suggesion: `generate JSS agent credentials for a cluster`

Reply:
I have updated this in 0acc3bb068bef8b517564899df5c394a5da0264e, but again used a different phrasing.